### PR TITLE
Update software-engineer-intern.md

### PR DESCRIPTION
**Pull request by mercadon**
Made some additional changes after our conversation last week

### CODE REVIEW

File: job-descriptions/software-engineer-intern.md [hunk 1]
```diff
@@ -16,17 +16,17 @@ Read [our blog](https://about.sourcegraph.com/blog/) to see what more we have be
...
-Requirements:
-- You are passionate about building tools for developers.
 
 Ideal candidates:
 
 - Are pursuing a computer science undergraduate or graduate degree.
-- Are familiar with one or more of the technologies that we use (e.g. Go, TypeScript, React).
-- Have built or contributed to a developer tool or library (e.g. https://github.com/sourcegraph/sourcegraph).
+- Are passionate about developer tools and have worked in the developer tool space in some capacity (built a developer tool, wrote a blog post, recorded a podcast, etc.)
+- Are active in the open ource community.
```
Comment (dadlerj) [inline job-descriptions/software-engineer-intern.md]: Missing an "s"

File: job-descriptions/software-engineer-intern.md [hunk 2]
```diff
@@ -16,17 +16,17 @@ Read [our blog](https://about.sourcegraph.com/blog/) to see what more we have be
...
+- Have submitted a PR in any one of our repos (and linked to it in their application)!
```
Comment (dadlerj) [inline job-descriptions/software-engineer-intern.md]: I might pose this as "Bonus: ..." or something? Is this too high of a bar @nicksnyder?

File: job-descriptions/software-engineer-intern.md (hunk 1 above)
Comment (dadlerj) [reply job-descriptions/software-engineer-intern.md]: Also, let's remove this bullet, and instead put it in the bullet above.

The one above could read:

- Can demonstrate passion and excitement about developer tools (e.g. have built a developer tool, contributed to open source developer tools or other open source projects, written blog posts or talks about development, etc.)

File: job-descriptions/software-engineer-intern.md (hunk 2 above)
Comment (mercadon) [reply job-descriptions/software-engineer-intern.md]: On the application it's listed as "optional"– so they'll still be able to submit their application regardless!

Comment (nicksnyder) [reply job-descriptions/software-engineer-intern.md]: Since we are talking about ideal I think it is ok to leave this here.

**Comment by nicksnyder** [comment]
Since this is just a PR into my branch, I am going to merge it into my PR and we can discuss there.

**Pull request merged**